src/main/java/io/xpring/xrpl/RippledFlags.java

```diff
@@ -0,0 +1,29 @@
+package io.xpring.xrpl;
+
+/**
+ * Flags used in ripppled transactions.
+ *
+ * Note: These are only flags which are utilized in Xpring SDK. For a complete list of flags, see:
+ * https://xrpl.org/transaction-common-fields.html#flags-field.
+ */
+public enum RippledFlags {
+    TF_PARTIAL_PAYMENT(131072);
```

Comment:
minor: Is it possible to express this in hex or binary?

`131072` is `0x00020000` (or 1 << 17) and it could be easier to see that the flag is just the 17th bit rather than a numerical value, per se.